scripts/globals/job_utils/geomancer.lua

```diff
@@ -52,9 +65,11 @@ xi.job_utils.geomancer.spawnLuopan = function(player, target, modelID, tickEffec
 
     -- Change the luopans appearance to match the effect
     -- TODO: This is should be the element of the spell being cast added as an offset
-    --       on top of a base model ID in core.
+    -- on top of a base model ID in core.
     luopan:setModelId(modelID)
 
+    -- Size is effected by EntityFlags
+    player:setPetFlags(luopanSize)
```

Comment:
technically, we should be setting just be using bitewise ops on its existing value just as a pattern for the folks who might copy paste from here for something else. (2nd and 3rd bits in the mask) rather then replacing the entire mask with a decimal value. But practically, it does not matter for this entity as it has not other flags, so I'm not going to insist, just noting it here in case it needs revisiting later or if someone disagrees with my dismissing that minor issue.

Reply:
I'd say just add a comment saying "NOTE: This is overwriting the whole value, rather than just setting individual flags"

Then copy/pasters have been warned

Reply:
I'm going to leave entity flags alone for now until a better solution / refactor has been put in place, as this i feel needs to be addressed before going further.

Reply:
No longer a problem, removed all the silly flag setting,
The flags dont seem to be setting correctly from the database so now they are set before spawn,
Props to Teo for helping with this.